subt_ign/worlds/finals_qual_demo.sdf

```diff
@@ -797,6 +797,78 @@
       <uri>https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Mine Cart</uri>
     </include>
 
+    <include>
+      <name>tile_83</name>
```

Comment:
`tile_83` exists already. I had to change this to `tile_87`

Reply:
Fixed in 3f4a2d5